mkosi/backend.py

```diff
@@ -214,6 +223,7 @@ class PartitionTable:
     first_lba: Optional[int]
 
     grain: int = 4096
+    _labelled: Dict[str, Partition] = dataclasses.field(default_factory=dict)
```

Comment:
wouldn't it be nicer to have an `Enum` for the partition types we care about instead of a string? that way python and the various tools should be better at finding typos with these ids?